coderd/database/queries/replicas.sql

```diff
@@ -18,19 +18,30 @@ INSERT INTO replicas (
 	"primary"
 ) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) RETURNING *;
 
--- name: UpdateReplica :one
-UPDATE replicas SET
+-- name: UpsertReplica :one
+INSERT INTO replicas (
+    id,
+    updated_at,
+    created_at,
+    started_at,
+    hostname,
```

Comment:
I think we should include `stopped_at` on the `INSERT` as well.  We'd expect it not to be set when a replica inserts itself the first time, but having an asymmetry between `INSERT` and `UPDATE` on an "upsert" can lead to confusing results.

For example, if one Coderd deletes a row, then another upserts it with `stopped_at` set to `now()`, we'd reinsert it without the `stopped_at`.